symbols.js

```diff
@@ -436,12 +436,24 @@ const symbols = [
     }
 ];
 
+let symbols = []
+
+try {
+    // Catching invallid json, not an invalid symbol array
+    symbols = JSON.parse(window.localStorage.getItem("symbols"));
+} catch(err) {
+    console.error(err);
+}
+if (!symbols) {
+    symbols = symbols_default;
+}
```

Comment:
I was thinking about this earlier today: what happens when the default symbols are updated? I think currently the user won't see them, they'll load what's in local storage and completely ignore the defaults.

We probably need some sort of merging. Alternately, we track users' custom symbols separately to the defaults.

Reply:
If you want to merge new defaults, with the existing symbols, we need to label either the user symbols or the defaults (which adds complexity) and then disable the editing of the defaults (which adds complexity) then it will be impossible for users to move the frequently used default symbols to the top of the list, resulting in users that will add existing default symbols also to the user symbols. Reducing the benefits of a list of default symbols.

Another method is to track all the changes made by the user (which adds way more complexity)  and apply them every time the defaults are loaded (which is so complex I cannot imagine how to do that)

If you can program that, you're welcome to do that. I probably don't have the knowledge, and if I did my time is also limited.

Another solution would be to publish sets of symbols, including a default. When a set (e.g. the default set) is updated let the user load that set into their collection. That's something I could implement in a few hours. As a matter of fact, I already did. It's in pull request #59